egs2/TEMPLATE/asr1/scripts/utils/table.csv

```diff
@@ -0,0 +1,112 @@
+corpus,task,name,url,fs,lang,gender,pytorch,espnet,commit,valid
```

Comment:
Is it copied from https://github.com/espnet/espnet_model_zoo/blob/master/espnet_model_zoo/table.csv?
How about directly using this instead of copying it?

Maybe, I misunderstood something.
Could you tell me where it is used?

Reply:
Hi Shinji, you are right. I could just have queried model downloader. I have made required changes.